server/block/slime_block.go

```diff
@@ -0,0 +1,39 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/world"
+)
+
+// SlimeBlock is a block that slows down entities and bounces them up if they drop onto it.
+type SlimeBlock struct {
+	solid
+	transparent
+}
+
+// EntityLand ...
+func (SlimeBlock) EntityLand(_ cube.Pos, _ *world.World, e world.Entity) {
+	if fallEntity, ok := e.(fallDistanceEntity); ok {
+		fallEntity.ResetFallDistance()
```

Comment:
I'm thinking we should probably do the proper velocity calculations here too, so that it also works for non-session entities.